src/p5.dance.interpreted.js

```diff
@@ -178,14 +178,31 @@ function whenPeak(range, func) {
 }
 
 /**
+ * Execute code at a specified time, either in measures or seconds.
+ * One idiosyncrasy: despite the function's name, if we call atTimestamp(4, "measures", foo),
+ * we actually fire on the 5th measure (i.e after 4 measures have completed).
  * @param {number} timestamp
  * @param {string} unit - Should be "measures" or "seconds"
  * @param {function} func - Code to run when event fires
  */
 function atTimestamp(timestamp, unit, func) {
-  // Despite the functions name, if we call atTimestamp(4, "measures", foo), we
-  // actually want to fire on the 5th measure (i.e after 4 measures have completed)
-  if (unit === 'measures') {
+  atTimestampEventGenerator(timestamp, unit, func, true);
```

Comment:
If you don't end up changing the name of this function, could you add a short comment letting reader know what `true` is assigned?
Something along the lines of: 
```
// true is assigned to `afterMeasure` in following call
atTimestampEventGenerator(timestamp, unit, func, true);
```

Reply:
I added docs to the new eventGenerator helper (incl. a description of the afterMeasure parameter), hopefully that clarifies

Reply:
Perfect - even better!